### DISCUSSION TITLE
Used custom block

### DISCUSSION THREAD

**Discussion by DevVigilante** [Sketchware-Pro/Sketchware-Pro#756] [Ideas]
Hey @JavkhlanK can you please add a feature in Sketchware pro that enable us to export used custom block in a project?
If I want to share our Sketchware pro with used custom block then I can create swb file then the used custom block are exported at that time but if someone other import my project to his device then the custom block will not be imported in his platte So please can you can add a feature to export used custom block separate from backup project as swb?

**Comment by DevVigilante** [comment]
I am saying about this 
![Screenshot_2023-02-10-14-55-00-889_com sketchware remod](https://user-images.githubusercontent.com/96680248/218055948-f1b38762-9e88-4190-9bc3-ef93d8046cbf.jpg)
Is my idea helpfull to others?
I am facing this problem because I transfer my project b/w 2 phones.

**Reply by j-px** [reply]
It definitely is!

**Comment by DevVigilante** [comment]
Anyone interested???
